neptunecontrib/monitoring/skopt.py

```diff
@@ -81,246 +95,95 @@ def log_results(results, experiment=None):
              results = skopt.forest_minimize(objective, space,
                                  base_estimator='ET', n_calls=100, n_random_starts=10)
 
-         Send best parameters to neptune::
-
-             import neptune
-             import neptunecontrib.monitoring.skopt as sk_utils
-
-             neptune.init(project_qualified_name='USER_NAME/PROJECT_NAME')
-
-             sk_utils.log_results(results)
-
-     """
-    log_best_score(results, experiment)
-    log_best_parameters(results, experiment)
-    log_plot_convergence(results, experiment, channel_name='diagnostics')
-    log_plot_evaluations(results, experiment, channel_name='diagnostics')
-    log_plot_objective(results, experiment, channel_name='diagnostics')
-
-
-def log_runs(results, experiment=None):
```

Comment:
Do you feel we don't need to be backward compatible here?

Reply:
I really don't think those functions were used separately. 
I can add deprecation warnings but I feel this is not necessary.

Reply:
Ack, I'm deferring to you on that.